src/Microsoft.ServiceFabric.Actors/Runtime/Migration/MigrationSettings.cs

```diff
@@ -0,0 +1,65 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace Microsoft.ServiceFabric.Actors.Runtime.Migration
+{
+    using System;
+    using System.Fabric;
+    using Microsoft.ServiceFabric.Actors.Generator;
+
+    internal class MigrationSettings
+    {
+        private static readonly string TraceType = typeof(MigrationSettings).ToString();
+
+        public MigrationSettings()
+        {
+            this.MigrationSourceOrchestratorNameSpace = "Microsoft.ServiceFabric.Actors.KVSToRCMigration.SourceMigrationOrchestrator, Microsoft.ServiceFabric.Actors.KVSToRCMigration";
+            this.MigrationTargetOrchestratorNameSpace = "Microsoft.ServiceFabric.Actors.KVSToRCMigration.TargetMigrationOrchestrator, Microsoft.ServiceFabric.Actors.KVSToRCMigration";
+            this.MigrationMode = MigrationMode.Auto;
+        }
+
+        public Uri SourceServiceUri { get; set; }
+
+        public Uri TargetServiceUri { get; set; }
+
+        public MigrationMode MigrationMode { get; set; }
+
+        internal string MigrationSourceOrchestratorNameSpace { get; set; }
+
+        internal string MigrationTargetOrchestratorNameSpace { get; set; }
+
+        internal virtual void LoadFrom(ICodePackageActivationContext codePackageActivationContext, string configSectionName = "MigrationConfig")
+        {
+            var configPackageName = ActorNameFormat.GetConfigPackageName();
+            try
+            {
+                var configPackageObj = codePackageActivationContext.GetConfigurationPackageObject(configPackageName);
+                if (configPackageObj.Settings.Sections.Contains(configSectionName))
```

Comment:
Should we add a handling here if the config section is not present in user settings.